server/integration/infinispan/src/main/java/org/jboss/as/clustering/infinispan/subsystem/CacheConfigurationAdd.java

```diff
@@ -865,6 +865,16 @@ public void inject(OutboundSocketBinding value) {
         }
     }
 
+   private void buildDbVersions(AbstractJdbcStoreConfigurationBuilder builder, OperationContext context, ModelNode store) throws OperationFailedException {
+      ModelNode dbMajorVersion = BaseJDBCStoreConfigurationResource.DB_MAJOR_VERSION.resolveModelAttribute(context, store);
```

Comment:
What is the advantage of using `resolveModelAttribute` over getting the value directly (`store.get`.. kind of operations)?